public/app/features/dashboard/components/SaveDashboard/SaveDashboardDiff.tsx

```diff
@@ -36,15 +54,17 @@ export const SaveDashboardDiff = ({ diff, oldValue, newValue }: SaveDashboardDif
     return <Spinner />;
   }
 
-  if (!value.diffs.length) {
+  if (value.count < 1) {
     return <div>No changes in this dashboard</div>;
   }
 
   return (
     <div>
-      <div className={styles.spacer}>{value.diffs}</div>
+      {value.schemaChange && <div className={styles.spacer}>{value.schemaChange}</div>}
+
+      {value.showDiffs && <div className={styles.spacer}>{value.diffs}</div>}
```

Comment:
Personally I think it should show a message like "Too many changes to show" when we've reached the threshold.